src/layouts/AllServices.tsx

```diff
@@ -0,0 +1,542 @@
+                        <Text component={TextVariants.a} href="/openshift/overview">
+                          OpenShift
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/openshift/releases">
+                          Releases
+                        </Text>
+                      </Text>
+                    </TextContent>
+                  </CardBody>
+                </Card>
+                <Card isPlain>
+                  <CardTitle>
+                    <BellIcon /> Integration and Notifications
+                  </CardTitle>
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>Alerts users to events, using email and integrations such as Webhooks.</Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
+                          Integration (Sources)
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
+                          Notifications
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
+                          Splunk | ServiceNow
+                        </Text>
+                      </Text>
+                    </TextContent>
+                  </CardBody>
+                </Card>
+                <Card isPlain>
+                  <CardTitle>
+                    <InfrastructureIcon /> Inventories
+                  </CardTitle>
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>
+                        View OpenShift clusters, Edge systems, RHEL hosts, and your organization&apos;s subscriptions.
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/openshift">
+                          Clusters
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/edge/fleet-management">
+                          Edge
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
+                          Subscriptions
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/insights/dashboard">
```

Comment:
This shouldn't navigate user to a dashboard, but rather to a inventory.

Reply:
@KendraMar